uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/CommonUtils.java

```diff
@@ -159,4 +159,11 @@ public static void copyFileToFolderInSdCard(final File file, @Nullable final Str
         AdbShellUtils.executeShellCommand("mkdir -p " + destinationPath);
         AdbShellUtils.executeShellCommandAsCurrentPackage("cp " + filePath + " " + destFilePath);
     }
+
+    public static void launchIntent(@NonNull final String action) {
```

Comment:
Since this is public, let's add some javadoc for it

Reply:
added the javadoc.